source/api-reference/usage.rst

```diff
@@ -1,12 +1,7 @@
 Usage API
 =========
 
-The Usage API provides an interface-agnostic way for QIIME 2 plugin
-developers to define examples of how to use their plugin’s actions. This
-enables the programmatic generation of examples for all QIIME 2 interfaces,
-eliminating the need to maintain specific examples for multiple interfaces.
-
-----
```

Comment:
I snuck is in after reassigning to you, @thermokarst (see https://github.com/qiime2/qiime2/pull/546/commits/d1dbf41b0b22b9058bf5b6c63caa4fdbe5c667b4)

Reply:
I think this should be put back where it was - please see https://github.com/qiime2/qiime2/pull/546